casino.gemspec

```
@@ -14,9 +14,8 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "~> #{File.read('.ruby-version').strip}"
 
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
+  s.files         = Dir.glob( 'lib/**/*.rb' ) + Dir.glob( 'db/**/*.rb' )
```

Do you need `app` as well? I think since the original was `git ls-files` (ie. _everything_) maybe this would give the result we want:
```suggestion
  s.files         = Dir.glob( '**/*' )
```

---

Unsure. Checking

---

Made some changes.
I'm not a fan of loading everything in the package. Because this definition of files-list exists for a reason, right?

---

Good point, approved 👍 

Btw you can save time by doing this instead :)

```ruby
Dir.glob( '{app,config,db,lib,public}/**/*.rb' )
```

---

That is a good one. 
Tested locally, it created the package well. 
Code updated.